### PR TITLE
Add missing files

### PR DESCRIPTION
**Pull request by hardik-codes**
Added missing language files in ```plugins```

### CODE REVIEW

File: plugins/actionlog/joomla/joomla.xml [hunk 1]
```diff
@@ -12,4 +12,8 @@
 	<files>
 		<filename plugin="joomla">joomla.php</filename>
 	</files>
+	<languages>
+		<language tag="en-GB">en-GB.plg_actionlog_joomla.ini</language>
+		<language tag="en-GB">en-GB.plg_actionlog_joomla.sys.ini</language>
+    	</languages>
```
Comment (infograf768) [inline plugins/actionlog/joomla/joomla.xml]: take off spaces and replace by tab

File: plugins/captcha/recaptcha_invisible/recaptcha_invisible.xml [hunk 2]
```diff
@@ -12,6 +12,10 @@
 	<files>
 		<filename plugin="recaptcha_invisible">recaptcha_invisible.php</filename>
 	</files>
+	<languages>
+		<language tag="en-GB">en-GB.plg_captcha_recaptcha_invisible.ini</language>
+		<language tag="en-GB">en-GB.plg_captcha_recaptcha_invisible.sys.ini</language>
+    	</languages>
```
Comment (infograf768) [inline plugins/captcha/recaptcha_invisible/recaptcha_invisible.xml]: take off spaces and replace by tab

File: plugins/actionlog/joomla/joomla.xml (hunk 1 above)
Comment (hardik-codes) [reply plugins/actionlog/joomla/joomla.xml]: @infograf768 In this I have used tabs only

**Comment by hardik-codes** [comment]
done @infograf768 
Please test

**Comment by infograf768** [comment]
OK now.

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on b8feb0131adc784955d624beff668d3f04efa3a0<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24453">issues.joomla.org/tracker/joomla-cms/24453</a>.</sub>

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on b8feb0131adc784955d624beff668d3f04efa3a0<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24453">issues.joomla.org/tracker/joomla-cms/24453</a>.</sub>

**Comment by infograf768** [comment]
rtc<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24453">issues.joomla.org/tracker/joomla-cms/24453</a>.</sub>

**Comment by HLeithner** [comment]
thx

**Pull request merged**